doc/guide/authentication.xml

```diff
@@ -99,4 +99,49 @@
 
     </section>
   </section>
+
+  <section id="2fa-auth">
+    <title>Enable two-factor authentication</title>
+
+    <para>As with any other <code>pam</pam> based service that supports it,
+      you can harden your Cockpit authentication by adding two-factor authentication.
```

Comment:
It might seem obvious that "2FA" means "OATH 6 digit codes", but maybe we should specifically state what kind of 2FA we're setting up here.  I find jumping direct from "2FA" to "install google-authenticator", without first specifying this detail, to be a bit jarring.

Reply:
You  are right, I should be more specific and I will use **TOTP/HOTP second factor authentication**